packages/malloy/src/lang/test/expressions.spec.ts

```diff
@@ -198,8 +198,8 @@ describe('expressions', () => {
       );
     });
     test('apply granular-literal alternation uses all literals for range', () => {
-      expect('ad ? @2020 | @2022').compilesTo(
-        '{{{ad >= @2020-01-01} and {ad < @2021-01-01}} or {{ad >= @2022-01-01} and {ad < @2023-01-01}}}'
+      expect('ad ? @2020').compilesTo(
+        '{{ad >= @2020-01-01} and {ad < @2021-01-01}}'
```

Comment:
Why is this test changing?

Reply:
Yup, at one point this was being generated as an IN and I was toying with different ways to address it and left the test change in, i'll put it back.

Reply:
however i changed it because the test was "does an apply turn into a range computation" not "does an apply of two range computations turn into an "IN". I was intending to add the second test and forgot